Milestone 4: Readme instructions

pipe2442 opened the pull request:
Milestone 4: Readme instructions

In this milestone we:

- Validated game info 
- added players name on each turn
- added README.txt info
- added Repl.it live demo

pipe2442 committed:
Fixed readme

pipe2442 committed:
Added replit repo

MahmoudBakr23 reviewed (changes requested):
### **Changes Required ♻🛑**

<hr>

Hello @pipe2442. 👋
Great work finishing this project all the way to the final milestone. Congratulations!👍🙌 However, there are still small changes you need to do before submitting🙏🙏

<hr>

- [ ] Please make your README.md file more descriptive (explain the game and how it is being played in case anyone could haven't see it before)
![prof_readme](https://user-images.githubusercontent.com/47625407/104537285-59c6a700-5622-11eb-83ab-e8f43571a428.png)

- [ ] Please make sure to inform the viewer how to run the game locally and how he play it in details
![readme](https://user-images.githubusercontent.com/47625407/104537262-503d3f00-5622-11eb-8303-a8842922ccde.png)

<hr>

Please make sure to do these changes before submitting for a re-review.
Good luck!

pipe2442 committed:
Update README.md

Added game instructions - added information on how to run the game locally - gave descriptive readme file.

pipe2442 commented:
Hi @MahmoudBakr23 the changes were updated as you requested, thanks for your advice!

ebeagusamuel reviewed (changes requested):
# Changes Required :recycle: 
### Hello,
#### Your readme is not yet descriptive enough. You need to be very descriptive with the instructions on how to play the game. Some people might not be familiar with the tictactoe game, so your readme should be set up in a way that anyone who reads the description there will be able to play the game.
### Request another review when you are done
slack(@ebeagusamuel)

pipe2442 committed:
Update README.md

Made readme file more descreptive.

pipe2442 committed:
Update README.md

Fixed readme

javitocor reviewed (approved):
# STATUS APPROVED 🥇 

Congratulations! 🥳 
You have met all the requirements! 👍 
Remember to merge it, fill up the completion form 📑  and you can move into the next project 🚀 
Good luck with it! 😃
